modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Blogs/BlogPostAdminController.cs

```diff
@@ -61,4 +61,12 @@ public virtual Task<BlogPostDto> UpdateAsync(Guid id, UpdateBlogPostDto input)
     {
         return BlogPostAdminAppService.UpdateAsync(id, input);
     }
+    
+    [HttpDelete]
+    [Route("removeCoverImage/{id}")]
```

Comment:
most of times, PascalCase naming is not a good approach in URLs, maybe we can do it as `remove-cover-image` or just a **DELETE** request to `{post-id}/cover-image/`

Reply:
The second looks better I think. `DELETE | .../blog-posts/123/cover-image` that request explains everything well.

what do you think?

```suggestion
    [Route("{id}/cover-image")]
```

Reply:
This changes requires re-generation of client-proxies